app/models/poll.rb

```diff
@@ -1,6 +1,7 @@
 class Poll < ActiveRecord::Base
   include Imageable
-
+  acts_as_paranoid column: :hidden_at
+  include ActsAsParanoidAliases
```

Comment:
These two lines, involve the Gem https://github.com/rubysherpas/paranoia
Which is used, to hide records from DB without actually erasing them.

Polls, do not need to have this functionality.
Please remove these two lines.

Reply:
These two lines are required by comments model, and while removing it, raises errors in polls comments claiming it needs hidden_at field.

Reply:
As stated in https://github.com/consul/consul/pull/1961#discussion_r142677847